src/app/public/src/stache.module.ts

```diff
@@ -42,6 +43,7 @@ export * from './modules/shared';
     StacheBackToTopModule,
     StacheBlockquoteModule,
     StacheBreadcrumbsModule,
+    StacheClipboardModule,
```

Comment:
I would not export the clipboard module to the public, since it's only being used internally by the code block.